app/src/main/java/ch/epfl/cs311/wanderwave/ui/components/login/SignInButton.kt

```diff
@@ -18,19 +19,23 @@ import androidx.compose.ui.res.painterResource
 import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.unit.dp
 import ch.epfl.cs311.wanderwave.R
+import ch.epfl.cs311.wanderwave.ui.theme.signInButtonColorPlaceHolder
 
 @Composable
 fun SignInButton(modifier: Modifier, onClick: () -> Unit) {
   Box(modifier = modifier.fillMaxWidth().padding(16.dp), contentAlignment = Alignment.Center) {
-    Button(onClick = { onClick() }, modifier = Modifier.testTag("signInButton")) {
-      Icon(
-          painter = painterResource(id = R.drawable.spotify_icon_rgb_black),
-          contentDescription = "Spotify Icon",
-          modifier = Modifier.height(27.dp))
-      Spacer(modifier = Modifier.width(12.dp))
-      Text(
-          text = stringResource(id = R.string.button_label),
-          style = MaterialTheme.typography.titleMedium)
-    }
+    Button(
+        onClick = { onClick() },
+        modifier = Modifier.testTag("signInButton"),
+        colors = ButtonDefaults.buttonColors(containerColor = signInButtonColorPlaceHolder)) {
```

Comment:
I would probably use Spotify green as button color, but it doesn't really matter if it's just a placeholder

Reply:
I tried Spotify green, not sure if I like it or not. I think we need a team decision.